# Every Seat, Every Game: observational evidence

The policy targets GA season-ticket holders. The club reports 23% fewer unused seats in that subgroup, but does not publish the subgroup fixture series needed for independent verification. The all-ticket series is a different population. Treat the raw contrast below as definition-sensitive.

In [1]:
from pathlib import Path
import sys, json
ROOT = Path.cwd()
if ROOT.name == 'notebooks': ROOT = ROOT.parent
sys.path.insert(0, str(ROOT))
import pandas as pd
from IPython.display import display, Image
d = pd.read_csv(ROOT / 'data/processed/fixtures.csv')
ctx = json.loads((ROOT / 'data/curated/context.json').read_text())


In [2]:
from analysis.policy import compare
p = compare(d)
display(pd.Series(p))

before_mean                                                          1465.5263
after_mean                                                           1516.8947
absolute_change                                                        51.3684
relative_change_pct                                                     3.5051
comparable_opponents                                                        16
paired_mean_change                                                      -1.625
paired_bootstrap_95                                          [-386.13, 366.94]
ga_st_reported_change_pct                                                  -23
caution                      Definition-sensitive descriptive contrast: 202...
dtype: object

In [3]:
paired = d[d.season.eq('2024-25')][['opponent','unused_tickets']].merge(d[d.season.eq('2025-26')][['opponent','unused_tickets']],on='opponent',suffixes=('_before','_after'),validate='one_to_one')
paired['difference']=paired.unused_tickets_after-paired.unused_tickets_before
display(paired.sort_values('difference'))

,opponent,unused_tickets_before,unused_tickets_after,difference
2,Bournemouth,2352,927,-1425
8,Manchester United,1977,569,-1408
0,Brentford,2262,1288,-974
14,Arsenal,1186,626,-560
1,Nottingham Forest,1679,1309,-370
6,Manchester City,948,867,-81
11,Everton,763,711,-52
9,Wolves,1208,1377,169
10,Newcastle United,1189,1367,178
3,Chelsea,1433,1654,221


## Interpretation

Sixteen common opponents have a mean contrast of −1.625 tickets; the opponent bootstrap interval crosses zero. This is not a causal interval. Season performance, schedules, overlapping ticketing policies, and source scope remain confounders. Obtain ticket-level gate entry and comparable unexposed groups before claiming an effect.

Sources and grain boundaries: [DATA_PROVENANCE.md](../DATA_PROVENANCE.md), [data contract](../docs/DATA_CONTRACT.md).